LibreSSL doesn't defined TLS1_3_VERSION so take route away from it

diff --git a/gsoap/stdsoap2.c b/gsoap/stdsoap2.c
index 91eff5b..efe647e 100644
--- gsoap/stdsoap2.c
+++ gsoap/stdsoap2.c
@@ -4394,7 +4394,7 @@ ssl_auth_init(struct soap *soap)
   /* enable all TSLv1 protocols and disable SSLv3 by default if no SSL/TLS flags are set */
   if (!(soap->ssl_flags & SOAP_SSLv3_TLSv1))
     soap->ssl_flags = SOAP_TLSv1;
-#if (OPENSSL_VERSION_NUMBER >= 0x10101000L)
+#if !defined(LIBRESSL_VERSION_NUMBER)
   if ((soap->ssl_flags & SOAP_SSLv3))
     minv = SSL3_VERSION;
   else if ((soap->ssl_flags & SOAP_TLSv1_0))
diff --git a/gsoap/stdsoap2.cpp b/gsoap/stdsoap2.cpp
index 102f007..f44ff05 100644
--- gsoap/stdsoap2.cpp
+++ gsoap/stdsoap2.cpp
@@ -4394,7 +4394,7 @@ ssl_auth_init(struct soap *soap)
   /* enable all TSLv1 protocols and disable SSLv3 by default if no SSL/TLS flags are set */
   if (!(soap->ssl_flags & SOAP_SSLv3_TLSv1))
     soap->ssl_flags = SOAP_TLSv1;
-#if (OPENSSL_VERSION_NUMBER >= 0x10101000L)
+#if !defined(LIBRESSL_VERSION_NUMBER)
   if ((soap->ssl_flags & SOAP_SSLv3))
     minv = SSL3_VERSION;
   else if ((soap->ssl_flags & SOAP_TLSv1_0))

